paket.dependencies

```diff
@@ -1,8 +1,8 @@
 version 5.198.0
 source https://api.nuget.org/v3/index.json
-frameworks: net46, netcoreapp3.1, netstandard2.0
```

Comment:
We still build design-time assembly for `net46`, please do not remove it.

`netstandard`-only version of TP nuget package is not ready yet.